### PR TITLE
Correct spelling/grammar in tutorial 10

### PR DESCRIPTION
**Pull request by hendrikmakait**
Fix typos and consistently use "VC dimension" instead of mix with "VC-dimension".

### CODE REVIEW

File: notes/10_svm/2_margin.tex [hunk 1]
```diff
@@ -265,7 +265,7 @@ \subsection{Margins and the VC-dimension}
          }
      \end{subfigure}
      \mode<article>{
-     \caption{A linear neuron can shatter any label configuration of 3 points regardless of how they are arranged in 2D space (excluding colinear points).}
+     \caption{A linear neuron can shatter any label configuration of 3 points regardless of how they are arranged in 2D space (excluding collinear points).}
```
Comment (kashefy) [inline notes/10_svm/2_margin.tex:268]: I thought you were going for AE spelling

**Review by kashefy — APPROVED** [review]

File: notes/10_svm/2_margin.tex (hunk 1 above)
Comment (hendrikmakait) [reply notes/10_svm/2_margin.tex:268]: My bad, didn't recognize this as a valid spelling. I only recalled that you had used "collinear" in previous documents.

Comment (kashefy) [reply notes/10_svm/2_margin.tex:268]: You're right, it's colinear here and collinear [here](https://github.com/kashefy/mi1notes/blob/master/notes/09_slt/1_linsep.tex#L35). So the latter would have to change to a single 'l' to be consistent with AE.

Comment (hendrikmakait) [reply notes/10_svm/2_margin.tex:268]: I will take care of that next time my brain hurts from studying.

Comment (kashefy) [reply notes/10_svm/2_margin.tex:268]: No worries, will take care of the collinear/colinear bit.

**Pull request merged**